collections/_posts/2023-12-20-CoCs.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: review
+title: "Image as Set of Points"
+tags: visual-representation clustering deep-learning
+author: "Robin Trombetta"
+cite:
+    authors: "Xu Ma, Yuqian Zhou, Huan Wang, Can Qin, Bin Sun, Chang Liu, Yun Fu"
+    title:   "Image as Set of Points"
+    venue: "International Conference on Learning Representations (ICLR) 2023 Oral"
+pdf: "https://arxiv.org/pdf/2303.01494.pdf"
+---
+
+# Highlights
+
+* The authors propose Context Clusters (CoCs) which is a new paradigm for visual representations where images are seen as a set of unorganized points.
+* Even though they don't achieve state-of-the-art performances, they are still on par with recent convolutional and transformer-based visual backbone.
+* CoCs have some good properties such as generalization to different data domains (point clouds, RGBD images, etc.) and good interpretability.
+* Code is available on this [GitHub repo](https://github.com/ma-xu/Context-Cluster).
+
+&nbsp;
+
+# Context Clusters
+
+To translate an image to a set of points, an input image $$ \textbf{I} \in \mathbb{R}^{3 \times w \times h}$$ is changed to an augmented collection of points (pixels) $$ \textbf{P} \in \mathbb{R}^{5 \times n}$$ with $$n = w \times h$$ and where each pixel $$\textbf{I}_{i,j}$$ has its RGB channel values enhanced with its 2D coordinates $$[\frac{i}{w} - 0.5 ; \frac{j}{h} - 0.5]$$ to give a vector of 5 numbers.
+
+Then, the features are extracted from this set of points with a succession of Points Reducers and Context Clusters blocks (Figure 1). As one of the goals of the model is to be plugged as a backbone feature extractor into more complex architecture, it copies the global architecture and intermediate/final output sizes of standard CNN and transformer-based backbones such as ResNet and ViTs. In particular, at each stage, the number of points is set so that it can be turned back into a square 2D image.
```

Comment:
Plural:
```suggestion
Then, the features are extracted from this set of points with a succession of Points Reducers and Context Clusters blocks (Figure 1). As one of the goals of the model is to be plugged as a backbone feature extractor into more complex architectures, it copies the global architecture and intermediate/final output sizes of standard CNN and transformer-based backbones such as ResNet and ViTs. In particular, at each stage, the number of points is set so that it can be turned back into a square 2D image.
```